src/Controls/src/Core/VisualElement.cs

```diff
@@ -920,7 +920,7 @@ internal void InvalidateStateTriggers(bool attach)
 
 		internal void MockBounds(Rectangle bounds)
 		{
-#if NETSTANDARD2_0
+#if NETSTANDARD2_0 || NET6_0
```

Comment:
Can we just delete this if/def? 

This was only here because of us building against NS1_0